client/my-sites/site-settings/disconnect-site/too-expensive.jsx

```diff
@@ -10,19 +9,17 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import CompactCard from 'components/card/compact';
-import Placeholder from 'my-sites/site-settings/placeholder';
-import QuerySitePlans from 'components/data/query-site-plans';
 import getCurrentPlanPurchaseId from 'state/selectors/get-current-plan-purchase-id';
-import { getSiteSlug } from 'state/sites/selectors';
-import { getSelectedSiteId } from 'state/ui/selectors';
+import QuerySitePlans from 'components/data/query-site-plans';
 import { addQueryArgs } from 'lib/url';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
 
 const TooExpensive = ( { confirmHref, planPurchaseId, siteId, siteSlug, translate } ) => {
-	if ( ! siteSlug || ! planPurchaseId ) {
+	if ( ! planPurchaseId ) {
 		return (
 			<div>
 				<QuerySitePlans siteId={ siteId } />
-				<Placeholder />
```

Comment:
The placeholder should still be displayed while the site plans are loading. Unlike the site itself, the plans are not guaranteed to be available and need to load first.

Reply:
This happens in such a rare case that I'm more than fine to remove the placeholder there - I did remove it intentionally. It is not beneficial right there IMHO, and just displaying nothing is more than fine. How do you feel about that?